backend/endpoints/rom.py

```diff
@@ -331,6 +331,7 @@ async def update_rom(
         )
         cleaned_data.update({"url_cover": ""})
     else:
+        cleaned_data["url_cover"] = data.get("url_cover", db_rom.url_cover)
```

Comment:
this won't work, you should update L340 with `url_cover=data.get("url_cover", db_rom.url_cover)`

Reply:
won't work in which terms? i tested it and it works when i want to update the cover or even when i just update some information of the rom 😕